internal/types/document_validation.go

```diff
@@ -38,6 +41,11 @@ func (e *ValidationError) Error() string {
 	return fmt.Sprintf("Invalid document, reason: %s.", e.reason)
 }
 
+// Unwrap returns unwrapped error.
+func (e *ValidationError) Unwrap() error {
```

Comment:
With `reason` not being exported and `Error() error` using `%s` instead of `%w`, `Unwrap() error` exposes internal details that were hidden before. Let's keep them hidden

Reply:
Removed that method.